dvc/command/diff.py

```diff
@@ -111,14 +111,15 @@ def _show_diff(diff, hide_missing=False):
         if not sum(summary.values()):
             return None
 
-        fmt = (
-            "files summary: {added} added, {deleted} deleted,"
-            " {renamed} renamed, {modified} modified"
-        )
-        if not hide_missing:
-            fmt += ", {not in cache} not in cache"
+        summary_message = "files summary: "
+
+        for state in states:
+            state_value = summary[state]
+            if state_value > 0:
+                summary_message += f"{state_value} {state}, "
 
-        ui.write(fmt.format_map(summary))
+        # Using [:-2] to remove last unnecessary ", " from message.
+        ui.write(summary_message[:-2])
```

Comment:
It would be better to just generate the message without the extra `, ` entirely so that it doesn't need to be removed like this (maybe consider using `", ".join(...)`)